cli/config/config.go

```diff
@@ -117,13 +119,71 @@ func InitViperConfig() {
 		_, err := AppFs.Stat(Path)
 		if err == nil {
 			log.Debug().Str("configfile", viper.ConfigFileUsed()).Msg("try to load local config file")
-			if err := viper.ReadInConfig(); err == nil {
-				LoadedConfig = true
+
+			// Check if this is a WIF config file
+			isWif, wifErr := IsWifConfigFormat(Path)
+			if wifErr != nil {
+				log.Debug().Err(wifErr).Msg("error checking for WIF config format")
+			} else if isWif {
+				log.Debug().Msg("detected WIF config file format")
+
+				// Convert the WIF config to Viper format
+				if err := ConvertWifConfig(Path, viper.GetViper()); err != nil {
+					LoadedConfig = false
+					log.Error().Err(err).Str("path", Path).Msg("could not convert WIF config file")
+				} else {
+					LoadedConfig = true
+					log.Debug().Msg("successfully converted WIF config")
+				}
 			} else {
-				LoadedConfig = false
-				log.Error().Err(err).Str("path", Path).Msg("could not read config file")
+				// Regular config file - load it normally
+				if err := viper.ReadInConfig(); err == nil {
+					LoadedConfig = true
+				} else {
+					LoadedConfig = false
+					log.Error().Err(err).Str("path", Path).Msg("could not read config file")
+				}
+			}
+		}
+	}
+
+	// Check if this is a WIF config file
+	// This detects the new format: {"type": "external_account", "audience": "...", "issuerUri": "..."}
+	if viper.GetString("type") == "external_account" {
+		log.Debug().Msg("detected WIF config format")
+
+		// Configure authentication method
+		if !viper.IsSet("auth") {
+			viper.Set("auth", map[string]string{"method": "wif"})
+		} else {
+			// If auth exists but method isn't set, set it to wif
+			authMap := viper.GetStringMap("auth")
+			if _, exists := authMap["method"]; !exists {
+				viper.Set("auth.method", "wif")
 			}
 		}
+
+		// Set the audience if available
+		if audience := viper.GetString("audience"); audience != "" {
+			viper.Set("audience", audience)
+		}
+
+		// Set the issuer URI if available
+		if issuerUri := viper.GetString("issuerUri"); issuerUri != "" {
+			viper.Set("issuer_uri", issuerUri)
+		}
+
+		// Set the API endpoint from universeDomain if available
+		if universeDomain := viper.GetString("universeDomain"); universeDomain != "" {
```

Comment:
Since all of these `viper.Get` function calls are happening before `viper.AutomaticEnv`, we won't have support for loading these options from environment variables.

If we want support, I would recommend moving them down below.

Reply:
Nice catch! Moved the `viper.Get` calls below `viper.AutomaticEnv` with my latest push.